samples/e2e/EventHubsCaptureEventGridDemo/Readme.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Readme
+description: An overview, how Event Hubs Capture integrates with Event Grid  
+services: event-hubs
+author: shubhavijayasarathy
+
```

Comment:
@ShubhaVijayasarathy This isn't being published on docs.microsoft.com, so you don't need the metadata section.